src/travel-details-screens/components/TripSection.tsx

```diff
@@ -220,6 +227,17 @@ const TripSection: React.FC<TripSectionProps> = ({
             />
           </TripRow>
         )}
+        {realtimeMapEnabled && onPressShowLive ? (
```

Comment:
Now I am on a very detailed level. It is not necessary to change, but more just a food for thought:

I think the check on `realtimeMapEnabled` can just be skipped here. It is good practice to be consistent on which component (parent or child) has the responsibility of sanity checking if something should be possible. As implemented here it is better if the parent has responsibility for sanity checking, and this component show the button as long as `onPressShowLive` is declared.